src/libraries/System.Text.Json/tests/Common/TestClasses/TestClasses.Constructor.cs

```diff
@@ -1972,6 +1973,61 @@ public void VerifyMinimal()
             MyTuple.Item4.Verify();
         }
     }
+    public class Class_With_Parameters_Default_Values
+    {
+        public int I { get; }
+        public float F { get; }
+        public double D { get; }
+        public decimal M { get; }
+        public StringComparison SC { get; }
+        public char C { get; }
+        public int? NI { get; }
+        public float? NF { get; }
+        public double? ND { get; }
+        public decimal? NM { get; }
+        public StringComparison? NSC { get; }
+        public char? NC { get; }
+
+        public Class_With_Parameters_Default_Values(
+                int i = 21, float f = 42.0f, double d = 3.14159, decimal m = 3.1415926535897932384626433M, StringComparison sc = StringComparison.Ordinal, char c = 'q',
```

Comment:
Should we add an `enum` here as well? To cover the case in https://github.com/dotnet/runtime/issues/94071.

Reply:
`StringComparison` is an enum.